docs/user-guide/lazy-loading.md

```diff
@@ -0,0 +1,94 @@
+# Lazy Loading
+
+> [!WARNING]
+> This is an experimental option and may not work as expected with all plugins.
+> The API may change without notice. > Please report any issues you encounter.
+
+> [!NOTE]
+> When the lazy loading functionality is enabled, the plugin's configuration is
+> routed to the enabled lazy provider. Options mentioned will be in relation to
+> the `${namespace}.${name}.lazyLoad` context.
+
+## Supported Lazy Loading Providers
+
+It is recommended to become familiar with lazy loading strategies supported by
+your provider in their upstream documentation and the corresponding plugin's
+support.
+
+Currently we only support **one** lazy loader:
+
+-lz.n: [`plugins.lz-n`](/plugins/lz-n/index.md) -
+[upstream docs](https://github.com/nvim-neorocks/lz.n?tab=readme-ov-file#plugin-spec)
```

Comment:
Not sure, but I believe this needs to stay on the same line, as list items are parsed differently to normal paragraphs.

Reply:
Turns out, this renders fine as-is.